src/main/java/com/iexec/core/feign/SmsClientWrapper.java

```diff
@@ -0,0 +1,42 @@
+package com.iexec.core.feign;
+
+import com.iexec.common.sms.SmsEnclaveChallengeResponse;
+
+import org.springframework.retry.annotation.Recover;
+import org.springframework.retry.annotation.Retryable;
+import org.springframework.stereotype.Service;
+
+import feign.FeignException;
+import lombok.extern.slf4j.Slf4j;
+
+
+@Slf4j
+@Service
+public class SmsClientWrapper {
```

Comment:
Isn't SmsService better than *Wrapper?

Reply:
Actually I thought we'd have a wrapper for each feign client so we handle failure, but if that generates a confusion I'll merge it with the service.